metadata/service/support.yml

```diff
@@ -0,0 +1,11 @@
+parameters:
+  watchdog:
+    _support:
+      collectd:
+        enabled: false
+      heka:
+        enabled: false
+      sensu:
+        enabled: false
```

Comment:
You ship sensu support so set this to true

Reply:
Fixed...